.eslintignore

```diff
@@ -1,3 +1,4 @@
 dist
 node_modules
 *.vue
+package.json
```

Comment:
Why ignore this?

Reply:
Because 
```json
{
  "exports": {
    ".": {
      "import": "./dist/index.mjs",
      "require": "./dist/index.js"
    }
  }
}
```

This will report an eslint error:

```
 error  Expected object keys to be in specified order. 'require' should be before 'import'  jsonc/sort-keys
```

Reply:
Yes, I know. Shouldn't we follow that order?

Reply:
update eslint to let import occur before require